src/System.Management.Automation/utils/ExecutionExceptions.cs

```diff
@@ -95,7 +95,7 @@ public CmdletInvocationException()
         /// <summary>
         /// Instantiates a new instance of the CmdletInvocationException class
         /// </summary>
-        /// <param name="message"> </param>
+        /// <param name="message"> .</param>
```

Comment:
Same as other one, does this actually reduce CodeFactor issues?  There's a bunch in this file.

Reply:
@SteveL-MSFT There already is a commit to remove this.

Reply:
You can ignore if it's already been resolved, was reviewing each commit in order